ui/src/app/utils/subscribable.spec.tsx

```diff
@@ -1,8 +1,4 @@
-import { atom } from './subscribable';
-
-export const delay = (ms) => {
```

Comment:
unrelated - heh, this test was not actually testing the `delay` from `subscribable`